homework/homework041223.py

```diff
@@ -0,0 +1,5 @@
+"""HOMEWORK1."""
+user_name = input('Введите Ваше имя: ')
```

Comment:
Try to use English phrases even for input. The will be no limitations for your code understanding  by others ( China, India, USA, Spain developers/qa etc)

Reply:
You're right!